cql3/Cql.g

```diff
@@ -71,9 +71,6 @@ options {
 #include "cql3/constants.hh"
 #include "cql3/operation_impl.hh"
 #include "cql3/error_listener.hh"
-#include "cql3/multi_column_relation.hh"
-#include "cql3/single_column_relation.hh"
-#include "cql3/token_relation.hh"
 #include "cql3/index_name.hh"
 #include "cql3/cql3_type.hh"
 #include "cql3/cf_name.hh"
```

Comment:
For the patch subject line: add 'for relations', since it already outputs expressions elsewhere.

Reply:
done